rust/rsc/src/rsc/read_job.rs

```diff
@@ -12,7 +12,7 @@ use std::collections::HashMap;
 use std::sync::Arc;
 use tracing;
 
-#[tracing::instrument]
+#[tracing::instrument(skip_all)]
 async fn record_use(job_id: Uuid, conn: Arc<DatabaseConnection>) {
```

Comment:
This basically tracks "job hits" right? Can we just skip `conn` and not the `job_id`?
Is it interesting to trace `resolve_blob(uuid)` also, to see blobs that get reused across jobs?